### ISSUE TITLE
Validation Dataset Unused in Tutorial?

### ISSUE THREAD

**Issue by dk683** [alexgkendall/SegNet-Tutorial#21]
Hello, @alexgkendall! Thank you for sharing the code.
I have looked at the SegNet tutorial and its corresponding model prototxt.
However, I have noticed that the validation dataset (val.txt) is not used.
Could you explain why? Also, is then the training accuracy based on the training dataset?
Thank you for your time! :-)

**Comment by alexgkendall** [comment]
Yes, the training accuracy is from the training set. Useful to look at how well the model fits the data (especially when you consider bayesian segnet's uncertainty). Absolutely, you can validate your model on the val.txt dataset, in the tutorial I don't bother with this for simplicity.

**Comment by dk683** [comment]
Aha! That perfectly makes sense. Thank you for your explanation!
